app/views/application/choices/find.html

```diff
@@ -11,6 +11,7 @@
 
 {% block primary %}
   <p class="govuk-body-l">Search for courses near you on Find postgraduate teacher&nbsp;training.</p>
+  <p class="govuk-body-l">A <a href="https://adviser-getintoteaching.education.gov.uk/">teacher training adviser</a> can help you understand your course options and support you with your application.</p>
```

Comment:
Does this sentence still look ok using the larger font (`govuk-body-l`) or should we drop it down to regular size?

Reply:
Good shout, looks much better as just body font!